Packs/SecurityScorecard/Integrations/SecurityScorecard/SecurityScorecard.py

```diff
@@ -0,0 +1,1078 @@
+    entries = portfolios.get('entries')
+
+    markdown = tableToMarkdown('Your SecurityScorecard Portfolios', entries, headers=['id', 'name', 'privacy'])
+
+    results = CommandResults(
+        readable_output=markdown,
+        outputs_prefix='SecurityScorecard.Portfolio',
+        outputs_key_field='id',
+        outputs=portfolios
+    )
+
+    return results
+
+
+def securityscorecard_portfolio_list_companies_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """`securityscorecard_portfolio_list_companies_command`: Retrieve all companies in portfolio.
+
+    https://securityscorecard.readme.io/reference#get_portfolios-portfolio-id-companies
+
+    :type ``client``: ``Client``
+    :type `` args``: ``Dict[str, Any]``
+        ``args['portfolio_id']``: Portfolio ID.
+            A list of Portfolio IDs can be retrieved using the `!securityscorecard-portfolios-list` command., type ``String``
+        ``args['grade']``: Grade filter. The acceptable values are capitalized letters between A-F, e.g. B., type ``String``
+        ``args['industry']``: Industry filter.
+            The acceptable values are capitalized, e.g. INFORMATION_SERVICES, TECHNOLOGY., type ``String``
+        ``args['vulnerability']``: Vulnerability filter, type ``String``
+        ``args['issue_type']``: Issue type filter, type ``String``
+        ``args['had_breach_within_last_days']``:
+            Domains with breaches in the last X days. Possible values are numbers, e.g. 1000., type ``Number``
+
+    :return:
+        A ``CommandResults`` object that is passed to ``return_results``
+    :rtype: ``CommandResults``
+    """
+
+    demisto.debug("securityscorecard_portfolio_list_companies_command called with args: {0}".format(args))
+
+    portfolio_id = str(args.get('portfolio_id'))
+
+    # Validate grade argument
+    if 'grade' in args:
+        grade = args.get('grade')
+    else:
+        grade = None
+
+    # Validate and transform industry
+    # We need to capitalize the industry to conform to API
+    industry = None
+    if 'industry' in args:
+        industry_arg = str(args.get('industry'))
+        industry = str.upper(industry_arg)
+    # else:
+    #     industry = None
+
+    vulnerability = args.get('vulnerability')
+
+    issue_type = args.get('issue_type')
+
+    had_breach_within_last_days = arg_to_number(
+        arg=args.get('had_breach_within_last_days'),
+        arg_name='had_breach_within_last_days',
+        required=False
+    )
+
+    response = client.get_companies_in_portfolio(
+        portfolio=portfolio_id,
+        grade=grade,
+        industry=industry,
+        vulnerability=vulnerability,
+        issue_type=issue_type,
+        had_breach_within_last_days=had_breach_within_last_days
+    )
+
+    # Check if the portfolio has more than 1 company
+    # Throw warning to UI if there are no companies
+    # str cast for mypy type incompatiblity
+    total_portfolios = int(str(response.get('total')))
```

Comment:
The `str` cast is because of [this](https://github.com/demisto/content/pull/13582#discussion_r672973432) comment. The `int` cast is required because the SecurityScorecard API requires the parameter to be of integer datatype.

Reply:
The str cast is not needed we can talk about how to fix mypy specifically. In general user arg_ti_number when you need to get an argument as an int.

Reply:
Resolved in commit 5a033dd1ff35c945823c949d0284fb25b63a8b63